dashboard/dashboard.html

```diff
@@ -130,26 +130,27 @@ <h4 id="building-city-address">San Francisco, CA 94103</h4> -->
 	        		<div class="row center">
 	        			<div class="col-xs-2"></div>
 	        			<div class="col-xs-8">
-	        			<div class="chart-title">ENERGY STAR SCORE FOR<span class="building-type"> HOTEL</span> BUILDINGS IN SAN FRANCISCO <span id="bulding-type-sq-ft"><br>150,000-200,000</span> SQUARE FEET</div>
+	        			<div class="chart-title">ENERGY STAR SCORE FOR<span class="building-type"> HOTEL</span> BUILDINGS IN SAN FRANCISCO<br><span id="bulding-type-sq-ft">150,000-200,000</span> SQUARE FEET</div>
 	        			</div>
 	        			<div class="col-xs-2"></div>
 	        		</div>
 	        		<div class="row center">
 			        	<div class="col-xs-3"></div>
 	        			<div class="col-xs-6">
+	        				<div class="loading">Loading&mdash;</div>
 	        				<div id="energy-star-score-histogram" class="histogram-chart"></div>
 	        			</div>
 	        			<div class="col-xs-3"></div>
 	        		</div>
 	        		<div class="row">
 			        	<div class="col-xs-12 panel-bottom">
-			            	<div class="panel-body bottom">
+			            	<!-- <div class="panel-body bottom"> -->
```

Comment:
I removed the bottom panel but wanted to see what Juliana thought before deleting the html, but I can do so before merging